service/controller/resource/monitoring/remotewriteagentsecret/test/case-1-awsconfig.golden

```diff
@@ -0,0 +1,14 @@
+data:
+  auth: YWxpY2U6ZW5jcnlwdGVk
+  password: cGFzc3dvcmQ=
+  username: YWxpY2U=
```

Comment:
again we do not see much what is being produced here, this does not bring much value.

Reply:
I'd rather have minimal tests than no tests at all

Reply:
But the test should produce output that we can compare against input in code.